apps/backoffice/.env

```diff
@@ -9,5 +9,6 @@ NEXT_PUBLIC_BACK_OFFICE_TOS_URL=https://io.selfcare.pagopa.it/tos.html
 NEXT_PUBLIC_BACK_OFFICE_PERS_DATA_PROTECTION_URL=https://privacyportal-de.onetrust.com/webform/77f17844-04c3-4969-a11d-462ee77acbe1/9ab6533d-be4a-482e-929a-0d8d2ab29df8
 NEXT_PUBLIC_BACK_OFFICE_OPERATIVE_MANUAL_URL=https://docs.pagopa.it/manuale-servizi/
 NEXT_PUBLIC_BACK_OFFICE_LOGIN_PATH=/auth/login
+NEXT_PUBLIC_BACK_OFFICE_IN_APP_PREVIEW_ENABLED=true
```

Comment:
It should be turned off for now, don't you think? 🙂
For test purpose you can set it **_true_** on your `.env.local` until development is complete.

Reply:
[0728218](https://github.com/pagopa/io-services-cms/pull/810/commits/07282188f55fa9a71852d5c5f39b91ffdf057c78)